### ISSUE TITLE
selected map marker from search results is not highlighted on the map

### ISSUE THREAD

**Issue by joachim-n** [organicmaps/organicmaps#7676]
⚠ Have you searched for similar, already existing issues?

**Describe the issue**

When I select a map marker from a search result to see more information about it, the map marker is not highlighted in any way, and so I can't tell which one I have information open for.

**Steps to reproduce**

1. Search using one of the built-in category searches, such as 'Where to eat' (These are REALLY useful btw!)
2. Lots of blue map markers appear on the map
3. Tap one marker to see more info about it
4. A window pops up with extra info - name, coordinates, bookmarks, etc, etc
5. There is now no way to tell which map marker the info window relates to. In the map has a lot of markers, or if I turn my attention somewhere else, I can't remember which item I am looking at.

**Expected behaviour**

In other cases, the tapped map marker is highlighted in blue:

- If I click on a map point for a shop without having done any search, 
- if I do a text search and then click on an item in the result list, 
- do a built-in category search and click on an item in the result list

The problem is that because the category search marks ALL its results with a blue highlight, the selected blue highlight is not visible.

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**System information:**

- Operating system and its version: iOS 15.8
-  Organic Maps version: 2024.03.07-1
- Device Model: iPad Air 2

**Additional context**
Please add any other context or comments here that may be useful.

**Comment by RedAuburn** [comment]
Looks like the icons are the same size as the blue highlight, so it covers it.
Either we should decrease the icon sizes, or make the highlight a bit bigger 👍️

**Comment by biodranik** [comment]
A screenshot would be helpful.

**Comment by joachim-n** [comment]
I didn't think there was much point since it's to show something that isn't there, but here is one:

![IMG_4273](https://github.com/organicmaps/organicmaps/assets/105262/5bcf593f-a038-46b1-b7ef-85d7d58e0462)

**Comment by Misalf-git** [comment]
It is there, if you believe in it.
![poi_search_selection.png](https://github.com/organicmaps/organicmaps/assets/72605561/e9f759b5-121f-4cfd-b94c-52d8adf596c6)

**Comment by RedAuburn** [comment]
Huh, on my (android) device it's not visible at all:

![ResizedImage_2024-03-26_11-21-04_2094.png](https://github.com/organicmaps/organicmaps/assets/26939824/3d4b65f2-59e4-47e0-8173-6b818bcd159c)

**Comment by Misalf-git** [comment]
Playing aroind with android's "**Display size**" setting, it seems the icons and the selection circle are scaling differently.

**Comment by vng** [comment]
I'd put high priority here.

@dvdmrtnz @RedAuburn @pastk Any ideas here?

**Comment by biodranik** [comment]
Use svg and recolor them dynamically?

**Comment by dvdmrtnz** [comment]
The simplest thing here would be making the selection circle bigger so it stands out from behind the search result icon

**Comment by RedAuburn** [comment]
How Google Maps (web) handles it:


https://github.com/user-attachments/assets/387e0bd1-5299-4461-945e-56112b72e676

**Comment by pastk** [comment]
I wonder why it wasn't an issue until recently? Search icons became bigger or are scaled differently now?

> The simplest thing here would be making the selection circle bigger so it stands out from behind the search result icon

Yeap it seems to be the easiest quick fix.
Or maybe use a bigger circle for search results only and leave a "regular" size for POIs, arbitrary points etc.

**Comment by biodranik** [comment]
> I wonder why it wasn't an issue until recently? Search icons became bigger or are scaled differently now?

Did https://github.com/organicmaps/organicmaps/pull/9448 increase search icons size? @dvdmrtnz

**Comment by dvdmrtnz** [comment]
> > I wonder why it wasn't an issue until recently? Search icons became bigger or are scaled differently now?
> 
> Did #9448 increase search icons size? @dvdmrtnz

No, it didn’t. Search result icons were the same same size before and after. Generated skin images can be checked to see that icons are the same size.

I refactored the search result icons to switch to svg format in PR https://github.com/organicmaps/organicmaps/ull/9227 in September 2024.
However, this issue was created in March 2024,  much earlier than that, so we have to look for any changes before that.

**Comment by biodranik** [comment]
But bookmark icon sizes somehow increased, right?

**Comment by dvdmrtnz** [comment]
I've built an old version to check the icon sizes and they are the same size. 

Here is version (2023.06.04-13 acc7c0547db4285dd8841ae7f98811268e38d908) vs current (2024.11.20-11⁩):

<img src="https://github.com/user-attachments/assets/2cfdb29d-c422-4f15-8c38-920e29f48335" width="320"/> <img src="https://github.com/user-attachments/assets/476377ec-0d8b-48f6-912d-abf687ee10c4" width="320"/>

---

Overlayed:

<img src="https://github.com/user-attachments/assets/b258c5e5-4343-4edc-b01f-af08003fbe87" width="320"/>

2023.06.04-13-android
